Documentation/freebsd-instructions.md

```diff
@@ -24,13 +24,35 @@ Install the following packages for the toolchain:
 - libunwind
 - gettext
 
-Note: LLDB is not yet available in the pkg repository on FreeBSD. The LLDB plugin is currently not being build on FreeBSD. Instructions for this will be added later.
-
 To install the packages you need:
 
-```janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake clang35 libunwind gettext```
+``janhenke@freebsd-frankfurt:~ sudo pkg install bash cmake clang35 libunwind gettext```
+
+You now have all the required components for building CoreCLR installed.
+
+Debugging CoreCLR (Optional)
+----------------------------
+
+Note: This step is *optional* and is not required to build CoreCLR itself. If you intend on hacking or debugging the CoreCLR source code, you need to follow these steps. You must follow these steps *before* starting the build itself.
+
+In order to debug CoreCLR you will also need to install LLDB, the LLVM debugger. LLDB is still in the process of being ported to FreeBSD, so no official packages exist (see the FreeBSD LLDB Wiki page for more information on what has been ported so far: https://wiki.freebsd.org/lldb). However, it is possible to manually download and install LLDB from the LLVM source tree by following the instructions below: 
+
+Firstly, install the following packages: python ninja swig13 git (in addition to the packages above), i.e.
+
+```janhenke@freebsd-frankfurt:~ % sudo pkg install python ninja swig13 git```
+
+Then, run the install script in ~/coreclr/src/pal/tools: 
```

Comment:
The instructions say assume `~/git/coreclr` and uses it in other places like `janhenke@freebsd-frankfurt:~/git/coreclr`. We should pick one or the other to stay consistent.